### PR TITLE
1.6 community release changelog

### PR DESCRIPTION
**Pull request by alenkacz**
JIRA issues: MARATHON-8128

### CODE REVIEW

**Comment by alenkacz** [comment]
thx for the help @meln1k

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
 ## Changes from 1.5.x to 1.6.0
 
+Recommended Mesos version is 1.5.0.
```
Comment (jeschkies) [inline changelog.md]: What does this mean?

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+As a part of performance improvement initiative we did a 10X improvement on the performance of the v2/groups endpoint [#5973](https://github.com/mesosphere/marathon/pull/5973)
+
+#### Maintenance mode support
+As of version 1.6, Marathon has simple built-in support for [Mesos Maintenance Primitives](http://mesos.apache.org/documentation/latest/maintenance/). For more informations please see our [docs](https://mesosphere.github.io/marathon/docs/maintenance-mode.html).
+
+#### Persistent volumes for pods
+Starting with this version it is possible to run stateful applications inside pods. For more informations and examples please see the [docs](https://mesosphere.github.io/marathon/docs/maintenance-mode.html).
+
+#### GPU preferential scheduling
+Marathon supports launching GPU tasks when the command-line option `--features gpu_resources` is specified. Depending on the GPU availability, it may be desirable to configure Marathon to avoid placing non-GPU tasks on GPU nodes. For more information see the [docs](https://mesosphere.github.io/marathon/docs/preferential-gpu-scheduling.html). or [MARATHON-8087](https://jira.mesosphere.com/browse/MARATHON-8087)
```
Comment (jeschkies) [inline changelog.md]: `https://mesosphere.github.io/marathon/docs/preferential-gpu-scheduling.html` give a 404. So we have to release the docs first, right?

File: changelog.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+### New versioning strategy
+As of 1.6 version, Marathon switched to the new versioning strategy. We still follow [SemVer](https://semver.org/) but we needed to have a unique version number for every master build. For that reason, first 1.6 public release is 1.6.322. It does not mean there were 321 releases prior to this one that we did not share with the community, the number actually is number of commits on the master branch since the last minor release (so 1.5). As for all future releases, the fix version will be increasing but don't expect the numbers to be consequential.
+
+### New features
+
+#### Performance improvements
+As a part of performance improvement initiative we did a 10X improvement on the performance of the v2/groups endpoint [#5973](https://github.com/mesosphere/marathon/pull/5973)
+
+#### Maintenance mode support
+As of version 1.6, Marathon has simple built-in support for [Mesos Maintenance Primitives](http://mesos.apache.org/documentation/latest/maintenance/). For more informations please see our [docs](https://mesosphere.github.io/marathon/docs/maintenance-mode.html).
```
Comment (jeschkies) [inline changelog.md]: `https://mesosphere.github.io/marathon/docs/maintenance-mode.html` gives a 404.

File: changelog.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
 
+Recommended Mesos version is 1.5.0.
+
+### New versioning strategy
+As of 1.6 version, Marathon switched to the new versioning strategy. We still follow [SemVer](https://semver.org/) but we needed to have a unique version number for every master build. For that reason, first 1.6 public release is 1.6.322. It does not mean there were 321 releases prior to this one that we did not share with the community, the number actually is number of commits on the master branch since the last minor release (so 1.5). As for all future releases, the fix version will be increasing but don't expect the numbers to be consequential.
+
+### New features
+
+#### Performance improvements
+As a part of performance improvement initiative we did a 10X improvement on the performance of the v2/groups endpoint [#5973](https://github.com/mesosphere/marathon/pull/5973)
```
Comment (jeschkies) [inline changelog.md]: Nit: `v2/groups`

File: changelog.md [hunk 5]
```diff
@@ -1,5 +1,35 @@
 ## Changes from 1.5.x to 1.6.0
```
Comment (jeschkies) [inline changelog.md]: Nit: to 1.6.322 or simply 1.6

File: changelog.md (hunk 1 above)
Comment (alenkacz) [reply changelog.md]: Exactly what it says :) The same thing is in Marathon 1.5 release notes (just with a different Mesos version)

File: changelog.md (hunk 2 above)
Comment (alenkacz) [reply changelog.md]: yes, this will work after we release docs...

File: changelog.md (hunk 3 above)
Comment (alenkacz) [reply changelog.md]: this will work after we release docs...

File: changelog.md [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+#### Maintenance mode support
+As of version 1.6, Marathon has simple built-in support for [Mesos Maintenance Primitives](http://mesos.apache.org/documentation/latest/maintenance/). For more informations please see our [docs](https://mesosphere.github.io/marathon/docs/maintenance-mode.html).
+
+#### Persistent volumes for pods
+Starting with this version it is possible to run stateful applications inside pods. For more informations and examples please see the [docs](https://mesosphere.github.io/marathon/docs/maintenance-mode.html).
+
+#### GPU preferential scheduling
+Marathon supports launching GPU tasks when the command-line option `--features gpu_resources` is specified. Depending on the GPU availability, it may be desirable to configure Marathon to avoid placing non-GPU tasks on GPU nodes. For more information see the [docs](https://mesosphere.github.io/marathon/docs/preferential-gpu-scheduling.html). or [MARATHON-8087](https://jira.mesosphere.com/browse/MARATHON-8087)
+
+#### And many more...
```
[the remark this answers is not in the document]
Comment (alenkacz) [reply changelog.md]: I used it instead of new features

File: changelog.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (alenkacz) [reply changelog.md]: thanks - btw. this is copypaste from our docs, so maybe we should change it there?

**Pull request merged**